integrations/amazon_bedrock/tests/test_chat_generator_utils.py

```diff
@@ -676,25 +682,18 @@ def test_callback(chunk: StreamingChunk):
                 meta={
                     "model": model,
                     "index": 0,
-                    "finish_reason": "tool_use",
+                    "finish_reason": "tool_calls",
                     "usage": {"prompt_tokens": 364, "completion_tokens": 71, "total_tokens": 435},
                     "reasoning_contents": [],
                 },
             )
         ]
 
+        # TODO Could update to check all streaming chunks
```

Comment:
do you want to do this in the current PR?

Reply:
Yes I do, I'll get on that